private/app/path/pathprobe/paths.go

```diff
@@ -186,7 +186,9 @@ func (p Prober) GetStatuses(ctx context.Context, paths []snet.Path,
 			continue
 		}
 		pathsPerIP[localIP.String()] = append(pathsPerIP[localIP.String()], path)
-		addStatus(PathKey(path), Status{Status: StatusTimeout, LocalIP: localIP})
+		//FIXME
+		//addStatus(PathKey(path), Status{Status: StatusTimeout, LocalIP: localIP})
+		addStatus(PathKey(path), Status{Status: StatusAlive, LocalIP: localIP})
```

Comment:
This appears to be the biggest hurdle currently: 

Figure out how to mark only Peering Links as `StatusAlive` by default (or some other status)

Reply:
False alarm, path probes appear to be working just fine across peering links